README.md

```diff
@@ -1,58 +1,55 @@
-# Purpose
+# .NET Authoring Tool for Extensions (Preview)
 
-There are two top level use-cases for needing .NET Core in VSCode:
+This extension allows acquisition of the .NET runtime specifically for VSCode extension authors. This tool is intended to be leveraged in extensions that are written in .NET and requires .NET to boot pieces of the extension (i.e. a language server). The extension is not intended to be used directly by users to install .NET for development.
 
-1. For extensions: A VSCode extension was written in .NET and requires .NET to boot pieces of the extension (i.e. a language server).
-1. For users: VSCode is the only IDE installed on the box and a user wants to use .NET for development.
+**This is a very early release of this tool. If you want to test it, reach out on [GitHub](https://github.com/dotnet/vscode-dotnet-runtime/issues) to discuss being in our early beta.**
 
-In this repos current state it only supports the first use-case but can easily be expanded to support the second.
+## Goals: Acquiring .NET Core for extensions
 
-## Acquiring .NET Core for extensions
-There are an increasing number of VS Code extensions that depend on .NET Core including Python, Live Share, Live Share Audio, Razor, Debug adapter, ARM JSON editor, Azure policy, F# and soon the Cloud Gateway extension.
+Extension authors do not know if the .NET core runtime is installed on the target machine. Existing solutions have a number of challenges:
 
-Today, extensions either ship or do dynamic acquisition of .NET Core on first activation. While it works, there are a number of challenges that have come up:
+1. **Duplication of .NET Core runtimes and slow updates**: Currently, each extension is acquiring its own copy of .NET core at ~30mb each.
+2. **Clean up**: When extensions install .NET Core in a non-VSCode folder location it is likely to be left behind.
+3. **Servicing and floating versions**: It is difficult to ensure that extensions will use the latest releases, particuarly without re-shipping.
+4. **Corrupted installations**: Corrupted installations can arise when VS Code is shut down mid-download or unzip.
+5. **Network security policies**: Alternative installation methods may result in errors due to blocking from network security policies.
+6. **Locked down envs**: Downloading a VSIX from the marketplace allows developers who are unable to freely install software to install extensions manually.
```

Comment:
This item feels like it is written in the positive. The rest negative. I think negative works here.